src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -337,6 +337,9 @@ private PreSubmitCallbackResponse<SscsCaseData> buildResponse(Callback<SscsCaseD
             }
             log.info("Saved the new interloc direction document for case id: " + caseData.getCcdCaseId());
         }
+
+        caseData.setPrePostHearing(null);
```

Comment:
Are we doing anything with this value before setting it as null here?

Reply:
nope, literally just meant to sit there and do nothing